easytab.h

```diff
@@ -870,21 +879,41 @@ EasyTabResult EasyTab_Load_Ex(HWND Window,
 
 EasyTabResult EasyTab_HandleEvent(HWND Window, UINT Message, LPARAM LParam, WPARAM WParam)
 {
+    static bool LeftContext = true;
     PACKET Packet = { 0 };
-
     if (Message == WT_PACKET &&
         (HCTX)LParam == EasyTab->Context &&
         EasyTab->WTPacket(EasyTab->Context, (UINT)WParam, &Packet))
     {
         POINT Point = { 0 };
         Point.x = Packet.pkX;
         Point.y = Packet.pkY;
-        ScreenToClient(Window, &Point);
+        if(LeftContext && EasyTab->IsRelative)
+        {
+            Point.x = 0;
+            Point.y = 0;
+            LeftContext = false;
+        }
+        if(!EasyTab->IsRelative)
+        {
+            ScreenToClient(Window, &Point);
+        }
         EasyTab->PosX = Point.x;
         EasyTab->PosY = Point.y;
-
+        EasyTab->Buttons = Packet.pkButtons;
```

Comment:
This is referring to whether specifically pkButtons is set to be returned as relative or absolute based on the flags set in `LogContext.lcPktMode` which is why I changed `#define PACKETMODE PK_BUTTONS` to `#define PACKETMODE 0` to set pkButtons to return as absolute values. 
> The pkButtons data item has different formats in absolute and relative modes, as determined by the PK_BUTTONS bit in the lcPktMode field of the context.

That said, if you can think of a more intuitive way of exposing it to the user go ahead =)